types/nicolaslopezj:roles/roles.d.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-restricted-globals */
+declare module 'meteor/nicolaslopezj:roles' {
+  type RuleFuncThis = { userId: string };
+  type RuleFunc = (this: RuleFuncThis, ...args: any[]) => boolean;
+  type Rule = RuleFunc | boolean
+
+  // eslint-disable-next-line import/prefer-default-export
+  export module Roles {
+    const Role: RoleStatic;
```

Comment:
This is a little shenanigans. There is a `class Foo` declaration style that would have worked better - something like:

```typescript
class Role {
  constructor(name: string);
  readonly name: string;
  function allow(action: string, func: Rule): void;
}
```

but eslint can't seem to handle a class declaration with methods with no body - it throws a parse error. It's fine with this nonsense, though (which I copied from the `meteor/mongo` declarations).